src/Compilers/Core/Portable/Diagnostic/Location.cs

```diff
@@ -29,6 +30,7 @@ internal Location()
         /// <summary>
         /// Returns true if the location represents a specific location in a source code file.
         /// </summary>
+        [MemberNotNullWhen(true, nameof(SourceTree))]
```

Comment:
Do we actually have this turned on yet? Or is just in anticipation?

Reply:
We build with /langver:preview and a new enough toolset so it's already active. I would have had to do suppressions otherwise.